### PR TITLE
Ticket/master/update all modules

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

**Comment by melissa** [comment]
I still need cron and scheduled_task, but those can happen in a seperate PR

File: configs/components/module-puppetlabs-mount_core.json [hunk 1]
```diff
@@ -1 +1 @@
-{"url":"git://github.com/puppetlabs/puppetlabs-mount_core.git","ref":"refs/tags/1.0.1"}
+{"url":"git://github.com/puppetlabs/puppetlabs-mount_core.git","ref":"refs/tags/1.0.2"}
```
Comment (jtappa) [inline configs/components/module-puppetlabs-mount_core.json]: The commit message says it's bumping this one to 1.0.1, was that a typo or was this one not supposed to be bumped twice?

Comment (melissa) [reply configs/components/module-puppetlabs-mount_core.json]: The commit message was probably a copy/pasta error. Thanks for catching that!

Comment (melissa) [reply configs/components/module-puppetlabs-mount_core.json]: It's been updated!

**Pull request merged**